src/Engine/Renderer/Material/BlinnPhongMaterial.hpp

```diff
@@ -8,12 +8,16 @@
 
 #include <Engine/Renderer/Material/Material.hpp>
 
-// just needed for struct TextureData
+// just need for struct TextureData
```

Comment:
? are you sure ;)
https://translate.google.com/#en/fr/just%20needed%20for%20struct%20TextureData

Reply:
Yes, I'm sure ... we only need to have access to the definition of TextureData

Reply:
But are you sure the tense is "need" and not "needed" ?
BTW I don't understand the comment, do you mean an include here has been removed or that

Reply:
I mean that this include is only necessary to access the definition of the struct TextureData. For this little thing, we bring a lot of others not needed ... BTW, we can remove this comment.